actions/setup-crib-environment/scripts/parse_profiles.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+profiles=$PROFILES
+
+if [ -n "$profiles" ]; then
```

Comment:
Nit, but since we're using bash anyway, I'd use double brackets where possible.

Reply:
fixed